priv/repo/migrations/20240626142058_add_check_execution_permissions.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Trento.Repo.Migrations.AddCheckExecutionPermissions do
+  use Ecto.Migration
+
+  def up do
+    execute "INSERT INTO abilities(id, name, resource, label, inserted_at, updated_at) VALUES (DEFAULT, 'all', 'cluster_checks_execution', 'Permits all operations on cluster checks executions', NOW(), NOW())"
+
+    execute "INSERT INTO abilities(id, name, resource, label, inserted_at, updated_at) VALUES (DEFAULT, 'all', 'host_checks_execution', 'Permits all operations on host checks executions', NOW(), NOW())"
```

Comment:
NOW(), NOW() is for inserted_at and updated_at?

Reply:
yes

Reply:
yes